core/base/Logging.h

```diff
@@ -139,8 +139,20 @@ inline void printLogT(_FmtType&& fmt, LogItem& item, _Types&&... args)
 #define AXLOG_WITH_LEVEL(level, fmtOrMsg, ...) \
     ax::printLogT(FMT_COMPILE("{}" fmtOrMsg "\n"), ax::preprocessLog(ax::LogItem{level}), ##__VA_ARGS__)
 
-#define AXLOGV(fmtOrMsg, ...) AXLOG_WITH_LEVEL(ax::LogLevel::Verbose, fmtOrMsg, ##__VA_ARGS__)
-#define AXLOGD(fmtOrMsg, ...) AXLOG_WITH_LEVEL(ax::LogLevel::Debug, fmtOrMsg, ##__VA_ARGS__)
+#if defined(_AX_DEBUG) && _AX_DEBUG > 0
```

Comment:
The `defined(_AX_DEBUG)` part is not necessary, it could be written as just `#if _AX_DEBUG > 0`. In this case if `_AX_DEBUG` is not defined it's converted to 0.

Reply:
Of course! Since it's still correct as-is, we can leave it in there until some other changes need to be done, and address it then.